applications/nrf_desktop/doc/fast_pair_app.rst

```diff
@@ -7,11 +7,11 @@ Fast Pair module
    :local:
    :depth: 2
 
-The Fast Pair module is used for following purpose:
+The Fast Pair module is used to:
```

Comment:
I would keep it as is so as to have a complete sentence.

Reply:
I adjusted it so that the grammar of the introduction sentence corresponds to the one applied in the bullet points below. We can keep the complete sentence but then I'd suggest to update the wording below